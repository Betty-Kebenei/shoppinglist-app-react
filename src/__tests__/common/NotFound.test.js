import React from 'react';
import { shallow, mount, render } from 'enzyme';

import Config from '../../Config';

import NotFound from '../../components/common/NotFound';

describe('interface of NotFound component', () => {
    it('should render', () => {
        const wrapper = shallow(<NotFound />);
        expect(wrapper).toMatchSnapshot();
    });
});