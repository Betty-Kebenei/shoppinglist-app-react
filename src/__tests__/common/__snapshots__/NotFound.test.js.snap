// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`interface of NotFound component should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          404!
        </h1>,
        <p>
          A page with that URL cannot be found.
        </p>,
      ],
      "className": "NotFound",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404!",
        },
        "ref": null,
        "rendered": "404!",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "A page with that URL cannot be found.",
        },
        "ref": null,
        "rendered": "A page with that URL cannot be found.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            404!
          </h1>,
          <p>
            A page with that URL cannot be found.
          </p>,
        ],
        "className": "NotFound",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404!",
          },
          "ref": null,
          "rendered": "404!",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "A page with that URL cannot be found.",
          },
          "ref": null,
          "rendered": "A page with that URL cannot be found.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
